test: rebranch migrate test from l-monninger/expose-types

andygolay opened the pull request:
# Summary
- **RFCs**: [Link to RFC](#./link/to/rfc), [Link to RFC](#./link/to/rfc), or $\emptyset$.
- **Categories**: any of `protocol-units`, `networks`, `scripts`, `util`, `cicd`, or `misc`.

<!--
Add your summary text here. 
 -->

# Changelog

<!-- 
Describe your changes. List roughly in order of importance.
-->

# Testing

<!--
Describe your Test Plan and explain added or modified test components.
-->

# Outstanding issues
<!--
List any outstanding issues that need to be addressed in future PRs, but which do not block merging this PR.
-->

l-monninger commented:
@andygolay what is the challenge you're having with the Docker overlays?

l-monninger commented:
@andygolay 

Ah, it took me a sec to realize what happened here. You hard-coded the `CONTAINER_REV` in, but it's not a release that's actually available. 

@nicolaslpf, switched us to short commit hashes a while back. So, you want: `7925499`

You can always check package tags here: https://github.com/movementlabsxyz/movement/pkgs/container/movement-full-node/versions

nicolaslpf commented:
Wrong nick

andygolay commented:
> @andygolay
> 
> Ah, it took me a sec to realize what happened here. You hard-coded the `CONTAINER_REV` in, but it's not a release that's actually available.
> 
> @mcmillennick , switched us to short commit hashes a while back. So, you want: `7925499`
> 
> You can always check package tags here: https://github.com/movementlabsxyz/movement/pkgs/container/movement-full-node/versions

I changed it to use an env var and then I run
```
CONTAINER_REV=b96f0fa docker compose -f docker/compose/movement-full-node/docker-compose.test-migrate-biarritz-rc1-to-pre-l1-merge.yml up -d --remove-orphans
```
and it errs with 
```
[+] Running 3/3
 ✘ Container movement-full-node-setup-1               Error                        0.6s 
 ✘ Container movement-full-node-movement-full-node-1  Error                        1.6s 
 ✔ Container celestia-light-node                      Created                      0.0s 
dependency failed to start: container movement-full-node-setup-1 exited (0)
```

The logs for `movement-full-node-setup-1` are simply
```
Setup complete
```
and the logs for `movement-full-node-movement-full-node-1` are
```
Attempting to bind metrics server to 127.0.0.1:9464
Metrics server successfully bound to 127.0.0.1:9464
Metrics server listening on 127.0.0.1:9464
Error: Failed to create the executor

Caused by:
    0: Failed to create the inner executor
    1: No db path provided.
```    

I don't see where I'm supposed to provide the db path, if it's the case that I should provide it as part of the environment.

l-monninger commented:
@andygolay What was the reason for closing this?

andygolay commented:
> @andygolay What was the reason for closing this?

@l-monninger both this and https://github.com/movementlabsxyz/movement/pull/1237 were me trying to create a docker compose overlay in the `movement` repo, to align with the approach in https://github.com/movementlabsxyz/movement-migration/pull/92

so as far as I can tell both this and 1237 should be closed, because we're taking the approach of using the embedded runners and lower-level Rust API.
